src/components/misc/personViews/PersonViewTableRow.jsx

```diff
@@ -7,8 +8,13 @@ import BooleanViewCell from './cells/BooleanViewCell';
 import PlainTextViewCell from './cells/PlainTextViewCell';
 import SurveyResponseCell from './cells/SurveyResponseCell';
 
+import {
+    addTagsToPerson,
+    removeTagFromPerson,
+} from '../../../actions/personTag';
 
-export default function PersonViewTableRow(props) {
+
+export default connect()(function PersonViewTableRow(props) {
```

Comment:
Is there a reason for not using the decorator here?

Reply:
I don't know why but I just assumed that a function declaration like this can't be "decorated". Don't know why I assumed that.

Reply:
I tried changing it now and it turns out my assumption was correct, at least with the version of babel that we are using. So I won't make any changes to this.